packages/twig/cypress/e2e/media/logo-grid.cy.js

```diff
@@ -22,4 +24,39 @@ describe("logo grid", () => {
         cy.get("img").should("have.attr", "src").and("not.be.empty");
       });
   });
+
+  it("at mobile breakpoint, shows items taking up full container width", () => {
+    cy.viewport("iphone-6");
+    cy.get(".ilo--logo-grid--logos").then(($container) => {
+      const containerWidth = $container.outerWidth();
+      cy.get(".ilo--logo-grid--item").each(($item) => {
+        const itemWidth = $item.outerWidth();
+        expect(itemWidth).to.equal(containerWidth);
+      });
+    });
+  });
+
+  it("at small breakpoint, shows items taking up half container width", () => {
+    cy.viewport("iphone-6+");
+    cy.get(".ilo--logo-grid--logos").then(($container) => {
+      const containerWidth = $container.outerWidth();
+      cy.get(".ilo--logo-grid--item").each(($item) => {
+        const itemWidth = $item.outerWidth();
+        // Item takes up half the width minus the grid gap
+        expect(itemWidth).to.equal(containerWidth / 2 - 16);
+      });
+    });
+  });
+
+  it("at large breakpoints ensure that there are 3 grid items in a row when there are 3 or more items.", () => {
+    cy.get(".ilo--logo-grid--logos").then(($container) => {
+      const containerWidth = $container.outerWidth();
+      cy.get(".ilo--logo-grid--item").should("have.length.gte", 3);
+      cy.get(".ilo--logo-grid--item").each(($item) => {
```

Comment:
Hey @Shashika6, this is one way to do it, but may be a more effective way to test:

1. That all of the items have the same width
2. That the width of all of the items is equal to 1/3 of the space available

Reply:
Sure @justintemps I'll also do a check to see if all the items have the same width

Reply:
May be that was clear, so first you get the width of the all of the items and makes sure it's the same.  Then you do one single check to make sure that the value is equal to 1/3 of the space available.

Btw I'm pretty sure there's a better way to assert that a value is equal to three other than by asserting it has to be at least three but less than four...

Reply:
Hey @justintemps this is the additional padding on the right I mentioned. 

![test](https://github.com/user-attachments/assets/6a7f0f4f-f545-4ef8-8392-3841f7954dfd)